helpers/lib/fonts.js

```diff
@@ -51,8 +51,10 @@ const fontProviders = {
             return collection;
         },
 
-        buildLink: function(fonts) {
-            return '<link href="https://fonts.googleapis.com/css?family=' + fonts.join('|') + '" rel="stylesheet">';
+        buildLink: function(fonts, fontDisplay) {
+            const displayTypes = ['auto', 'block', 'swap', 'fallback', 'optional'];
+            fontDisplay = displayTypes.includes(fontDisplay) ? fontDisplay : null;
```

Comment:
Would be simpler if you just made `swap` the default value here. Then you wouldn't need an additional conditional in the string construction on the next line.

```
fontDisplay = displayTypes.includes(fontDisplay) ? fontDisplay : 'swap';
return `<link href="https://fonts.googleapis.com/css?family=${fonts.join('|')}&display=${fontDisplay}" rel="stylesheet">`;
```

Reply:
Refactored logic.